tests/flexmock_test.py

```diff
@@ -1,10 +1,19 @@
 """Flexmock tests."""
-# pylint: disable=missing-docstring,too-many-lines,disallowed-name,no-member,invalid-name,no-self-use
+# pylint: disable=missing-docstring
+# pylint: disable=protected-access
+# pylint: disable=too-many-lines
+# pylint: disable=disallowed-name
+# pylint: disable=no-member
+# pylint: disable=invalid-name
+# pylint: disable=no-self-use
+# pylint: disable=too-few-public-methods
 import random
 import re
 import sys
 import unittest
 
+import pytest
```

Comment:
Don't use pytest here, I believe the point of this test is to use regular `unittest`. And seriously it's not so bad style-wise either, they are maintainable.

Reply:
I was thinking about that too. I wonder if we should run tests with other test frameworks so that we only install that one test framework and not others.

Reply:
Makes sense, I'll revert the change here.